apps/application-system/api/src/app/modules/application/application.controller.ts

```diff
@@ -38,13 +38,13 @@ import {
   ApplicationStatus,
   CustomTemplateFindQuery,
 } from '@island.is/application/core'
-import { Unwrap, Locale } from '@island.is/shared/types'
+import type { Unwrap, Locale } from '@island.is/shared/types'
```

Comment:
There is this rule available for import types https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/consistent-type-imports.md I guess it would be nice to enable it for the whole repo

Reply:
Sounds really good. The changes needed for that are huge though, so I'd like to create a separate issue for that.

Reply:
https://github.com/island-is/island.is/issues/4001

Reply:
I agree. Maybe it could start for each project's tsconfig that are using the esbuild. That should already be done, shouldn't it? so we could graduely introduce it and have it already enabled so we won't break esbuild for the existing projects using it?

Reply:
That would make sense. It's a mundane boring task of adding it - I'll get right on it (on another branch first to test it)

Reply:
As seen in https://github.com/island-is/island.is/actions/runs/885666948, there seems to be a lot of cases that the linter catches but esbuild doesn´t care about. I'm abandoning this for now.